### PR TITLE
Cleanup fixture reading in extended tests

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @mfojtik 
@openshift/sig-master

### CODE REVIEW

File: test/extended/util/manifests.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+	obj, _, err := scheme.Codecs.UniversalDeserializer().Decode(data, nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return obj, nil
+}
+
+func ReadFixtureOrFail(path string) runtime.Object {
```
Comment (soltysh) [inline test/extended/util/manifests.go]: `ReadFixtureOrDie` is the preferred name for these methods.

Comment (tnozicka) [reply test/extended/util/manifests.go]: thought that's the naming for cases when it raises `panic` this one just fails the tests

Comment (soltysh) [reply test/extended/util/manifests.go]: In tests, that's basically that.

**Pull request merged**